shell/platform/android/BUILD.gn

```diff
@@ -652,3 +652,35 @@ action("embedding_jars") {
     }
   }
 }
+
+# Renames android artifacts and places them in the final
+# expected folder structure.
+action("abi_jars") {
+  script = "//flutter/build/android_artifacts.py"
+  deps = [
+    ":android_jar",
+    ":pom_libflutter",
+  ]
+
+  artifact_id =
+      string_replace(android_app_abi, "-", "_") + "_" + flutter_runtime_mode
+  sources = [
+    "$root_out_dir/${artifact_id}.jar",
```

Comment:
See https://gn.googlesource.com/gn/+/master/docs/reference.md#func_get_target_outputs.

And to filter down to the outputs that you need, you can use https://gn.googlesource.com/gn/+/master/docs/reference.md#func_filter_include